sources/Runner/ScoreScene.cpp

```diff
@@ -58,17 +58,17 @@ namespace BBM
 		scene->addEntity("scene title text")
 			.addComponent<PositionComponent>(1920 / 2.37, 250, 0)
 			.addComponent<Drawable2DComponent, RAY2D::Text>("CONGRATS", 50, RAY::Vector2(), ORANGE);
-		for (int i = 0; i < players.size(); i++) {
-			auto &playerTile = scene->addEntity("player tile")
+		for (int i = 0; i < static_cast<int>(players.size()); i++) {
```

Comment:
The real fix was to use a std::size_t for i instead of silencing this warning like that.
(we don't really care but anyway)

Reply:
fixed anyway, you're right, it's not very smart :/